modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -25,10 +28,15 @@ spec:
   template:
     metadata:
       annotations:
-        descheduler.alpha.kubernetes.io/evict: "true"
+        descheduler.alpha.kubernetes.io/prefer-no-eviction: "true"
 ----
 
-. Configure the `KubeDescheduler` object with the `LongLifecycle` profile and enable background evictions for improved VM eviction stability during live migration:
+. Configure the `KubeDescheduler` object with the `KubeVirtRelieveAndMigrate` profile and enable background evictions for improved VM eviction stability during live migration:
++
+[NOTE]
+====
+When using the `prefer-no-eviction` annotation to exclude VMs, the `KubeVirtRelieveAndMigrate` profile is sufficient for VM evictions. Do not enable `EvictPodsWithLocalStorage` or `EvictPodsWithPVC`.
```

Comment:
This note sounds confusing to me.
`KubeVirtRelieveAndMigrate` profile is sufficient by itself for VM evictions even if the user never sets the `prefer-no-eviction` annotation (that can be optionally used to get a few selected/critical VMs ignored by the descheduler).
I'd also remove the part about `EvictPodsWithLocalStorage` or `EvictPodsWithPVC` becouse those two options are simply implicitly enabled as a result of the `KubeVirtRelieveAndMigrate` profile.